src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -10,53 +10,61 @@ public enum BusChain {
 	ROBORIO("rio"),
 	CANIVORE("CANivore");
 
+	private static final double MAX_CAN_UTILIZATION_PERCENT = 0.6;
+	private static final double MAX_RECEIVE_ERRORS = 0;
+	private static final double MAX_TRANSMIT_ERRORS = 0;
+	private static final String LOG_PATH = "Bus/";
+
 	private final String chainName;
+	private final String logPath;
+	private CANBus.CANBusStatus busStatus;
 
 	BusChain(String chainName) {
 		this.chainName = chainName;
+		this.logPath = LOG_PATH + getChainName();
+		this.busStatus = CANBus.getStatus(getChainName());
```

Comment:
Why getter?

Reply:
@YHerm ?

Reply:
@drorta ?

Reply:
I think listen to yoni can bi the right choice! @YoniKiriaty what do you think?

Reply:
I think @YoniKiriaty should know why there is a getter instead of using the variable

Reply:
let me think for a bit
![image](https://github.com/user-attachments/assets/fdfb0d91-9f76-4d7e-94ef-713edee1ce4e)

Reply:
nah idk maybe @YHerm  knows

Reply:
I really don't know. I think we should ask @drorta